website/src/pages/documents/document.css.ts

```diff
@@ -224,3 +224,22 @@ export const mobileWordPanel = style([
     },
   },
 ])
+
+export const scrollTop = style({
+  display: "flex",
+  flexDirection: "row",
+  flexWrap: "wrap",
+  flexFlow: "column nowrap",
+  justifyContent: "flex-end",
+  alignContent: "center",
+  alignItems: "center",
+  alignSelf: "flex-end",
+  position: "sticky",
+  zIndex: 1,
+  top: `calc(100vh - 105px)`,
```

Comment:
Is there a difference between this and using `bottom: 105`?

Reply:
Yes I tried that before, but for some reason it doesn't position right and also stops being sticky as well. I can't exactly figure out why.